Remove parent nucleus-parent for docs pom.xml to fix the build

OndroMih opened the pull request:
To fix the Jenkins build job for docs, which gives "Could not find artifact org.glassfish.main:nucleus-parent:zip:resources:7.0.0-SNAPSHOT": https://ci.eclipse.org/glassfish/view/all/job/glassfish-docs-publish/507/console

Nothing from nucleus-parent is needed to build the docs and the parent isn't built together with the docs module, giving an error in the Jenkins job.

This PR also adds the missing news about the 6.2.3 release to the website.

Commit:
Remove parent nucleus-parent for docs pom.xml

To fix the Jenkins build job for docs, which gives "Could not find artifact org.glassfish.main:nucleus-parent:zip:resources:7.0.0-SNAPSHOT".
Nothing from nucleus-parent is needed to build the docs and the parent isn't built together with the docs module, giving an error in the Jenkins job.

Commit:
Website: Add news about the 6.2.3 release

dmatej reviewed (changes requested):
If you change parents you have to add the plugin-management and define missing plugin versions. It seems it is just two plugins, so it should be simple.

```
[WARNING] Some problems were encountered while building the effective model for org.glassfish.docs:add-on-component-development-guide:jar:7.0.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-jar-plugin is missing. @ org.glassfish.docs:parent:7.0.0-SNAPSHOT, /media/data-samsung/data/dmatej/work/repo/git/glassfish-beta/docs/parent/pom.xml, line 183, column 21
[WARNING] 'build.plugins.plugin.version' for org.codehaus.mojo:build-helper-maven-plugin is missing. @ org.glassfish.docs:parent:7.0.0-SNAPSHOT, /media/data-samsung/data/dmatej/work/repo/git/glassfish-beta/docs/parent/pom.xml, line 162, column 21
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.glassfish.docs:administration-guide:jar:7.0.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-jar-plugin is missing. @ org.glassfish.docs:parent:7.0.0-SNAPSHOT, /media/data-samsung/data/dmatej/work/repo/git/glassfish-beta/docs/parent/pom.xml, line 183, column 21
[WARNING] 'build.plugins.plugin.version' for org.codehaus.mojo:build-helper-maven-plugin is missing. @ org.glassfish.docs:parent:7.0.0-SNAPSHOT, /media/data-samsung/data/dmatej/work/repo/git/glassfish-beta/docs/parent/pom.xml, line 162, column 21
```

Commit:
Add missing versions for plugins in the docs parent

After the nucleus-parent that defined them in plaugin management was removed as a parent.
